### PR TITLE
PAINTROID-719: Use `.fvmrc` config file as `fvm_config.json` is deprecated

### PR DESCRIPTION
**Pull request by bhav-khurana**
## Ticket
[PAINTROID-719](https://catrobat.atlassian.net/browse/PAINTROID-719)
## New Features and Enhancements
- Create the `.fvmrc` config file
- Add `.fvm/` directory to gitignore

#### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

[PAINTROID-719]: https://catrobat.atlassian.net/browse/PAINTROID-719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by msesko** [comment]
Please remove the fvm_config.json from the repo. Just putting ./fmv folder in gitignore wont delete it.

**Review by msesko — CHANGES_REQUESTED** [review]
Please remove the fvm_config.json from the repo. Just putting ./fmv folder in gitignore wont delete it.

**Review by msesko — APPROVED** [review]
Good

**Comment by juliajulie95** [comment]
After installing the new fvm Version and using "fvm install" I get this error:
```
SDK Version : 3.10.5 has Dart SDK 3.0.5 does not meet the project constraints of >=2.17.0 <3.0.0.
This could cause unexpected behavior or issues.
```

I think we should fix that immedeatly.
Please add to your PR so we can get this merged quickly :)
I'm not exactly sure but maybe we can use 
```
environment:
  sdk: ">=3.0.0 <4.0.0"
```

**Comment by juliajulie95** [comment]
Oh maybe not, because then I gett errors in the code.
I already created a ticket for flutter update, I'll add it there

**Comment by juliajulie95** [comment]
I also now get the error in path_with_action_history.dart.
Even though the flutter version did not change.
Do you have the same?
`Missing concrete implementations of 'Path.addArc', 'Path.addOval', 'Path.addPath', 'Path.addPolygon', and 22 more. `

**Comment by bhav-khurana** [comment]
> ```
> environment:
>   sdk: ">=3.0.0 <4.0.0"
> ```

I think changing it to `sdk: ">=2.17.0 <3.1.0"` should do the work

**Comment by juliajulie95** [comment]
Please add that then :)

File: pubspec.lock [hunk 1]
```diff
@@ -1521,5 +1529,5 @@ packages:
     source: hosted
     version: "2.1.1"
 sdks:
-  dart: ">=3.0.5 <4.0.0"
+  dart: ">=3.2.0-194.0.dev <4.0.0"
```
Comment (juliajulie95) [inline pubspec.lock:1532]: This cannot be right I think.
When I do fvm flutter pub get it sets this to dart: ">=3.0.5 <3.1.0" which also aligns with what you wrote in the pubspec.yaml file.
Did you update this file too after changing the pubspec.yaml?

**Comment by juliajulie95** [comment]
Please resolve conflicts

**Comment by juliajulie95** [comment]
changed in another PR

**Pull request closed without merging**